Android/quickstart-java/.idea/misc.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="1.8" project-jdk-type="JavaSDK">

```

Comment:
Just curious -- should the value for languageLevel match the project-jdk-name?

Reply:
These are the default of the bootstrapped project that should work on any version of Android. Leaving as-is.